scripts/utils/v4X_compatibility_test.sh

```diff
@@ -3,13 +3,14 @@
 # Checkout 4.X integration test directory
```

Comment:
It might be good to just put

```bash
set -xe
```

at the top of this script, that way if any one of these commands fails, the whole script fails with a non-zero exit code